### DISCUSSION TITLE
[A-L] (9 Settembre 2021) Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#324] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152791894-e226f384-eacd-4ae8-badb-dcb2d45dca44.png)

**Comment by GattoBrizzolato** [comment]
![es6hbfhdr](https://user-images.githubusercontent.com/98970164/171063761-2ae18848-a05d-4a84-b052-505c366bec81.jpeg)

**Comment by Near047** [comment]
e se a e b fossero a = -1 e b = 3, allora sempre nel caso base si potrebbe avere anche 0, giusto ?

**Comment by GattoBrizzolato** [comment]
In questo caso specifico 'a' non può essere uguale a -1 in quanto  sia 'a' che 'b' appartengono all'insieme dei NUMERI NATURALI

**Comment by LuginiAndrea** [comment]
Dimostrazione alterntiva Passo induttivo:
Sappiamo che $n = 3a + b$ per induzione, il che implica che $n + 1 = 3a + b + 1$. Dividiamo in due casi per $b$:

1. $0 \leq b < 2$: ne segue che $1 \leq b + 1 < 3$, quindi $n + 1$ è inscrivibile nella forma voluta
2. $b = 2$: ne segue che $b + 1 = 3 \implies n + 1 = 3a + 3 = 3\left(a + 1\right) + 0$, quindi $n+1$ è inscrivibile nella forma voluta
